Modules/pages.js

```diff
@@ -0,0 +1,57 @@
+import { renderBooks } from './renderBooks.js';
+
+const itemList = document.getElementById('li-list');
+const itemAdd = document.getElementById('li-add');
+const itemContact = document.getElementById('li-contact');
+
+const bookListDiv = document.getElementById('table');
+const formDiv = document.getElementById('form');
+const contactDiv = document.getElementById('contact');
+
+function constructListPage() {
+  bookListDiv.style.display = 'block';
+  formDiv.style.display = 'none';
+  contactDiv.style.display = 'none';
+
+  renderBooks();
+}
+
+function constructFormPage() {
+  bookListDiv.style.display = 'none';
+  formDiv.style.display = 'block';
+  contactDiv.style.display = 'none';
+}
+
+function constructContactPage() {
+  bookListDiv.style.display = 'none';
+  formDiv.style.display = 'none';
+  contactDiv.style.display = 'block';
+}
+
+export default function page(page) {
+  switch (page) {
```

Comment:
- [ ] as mentioned in the requirements you should Use ES6 syntax, this includes making use of arrow functions instead of the `function` keyword.